src/commands/test/lib/error-info-builder.ts

```diff
@@ -6,7 +6,7 @@ export function buildErrorInfo(err: any, profile: Profile, command: AppCommand):
   const exitCode = err.exitCode || err.errorCode || ErrorCodes.Exception;
   let message : string = null;
 
-  let helpMessage = `Further error details: For help, please send both the reported error above and the following environment information to us by going to https://appcenter.ms/apps and starting a new conversation (using the icon in the bottom right corner of the screen)${os.EOL}
+  let helpMessage = `Further error details: For help, please send both the reported error above and the following environment information to us by going to https://appcenter.ms/apps and open a support ticket (using the "?" menu in the top right corner of the screen)${os.EOL}
```

Comment:
open -> opening (since it says 'going' earlier)
and top -> upper

Reply:
@anywherepilot Thanks! Do you mind elaborating on the difference between 'and top' and 'upper'?

Reply:
I should use quotes ^^ I meant and "top" -> "upper". You already changed it.